docs/user-guide/at-tls-configuration.md

```diff
@@ -1,23 +1,642 @@
+LTPA token and SAF provider cannot be used in this configuration because API ML cannot access the hardware key to sign its own tokens.
+
+## AT-TLS rules
+
+This section describes suggested AT-TLS settings, and serves as guidelines to set your AT-TLS rules.
+
+### Inbound rules
+
+A generic inbound rule can be set for all Zowe services:
+
+```bash
+TTLSRule ZoweServerRule
+{
+  LocalAddr All
+  RemoteAddr All
+  LocalPortRange 7551-7559 # Range covers all Zowe services
+  Jobname ZWE1* # Jobname according to zowe.job.prefix in zowe.yaml
+  Direction Inbound
+  TTLSGroupActionRef ServerGroupAction
+  TTLSEnvironmentActionRef ZoweServerEnvironmentAction
+  TTLSConnectionActionRef ZoweServerConnectionAction
+}
+
+TTLSGroupAction ServerGroupAction
+{
+  TTLSEnabled On
+}
+
+TTLSEnvironmentAction ZoweServerEnvironmentAction
+{
+  HandshakeRole ServerWithClientAuth
+  EnvironmentUserInstance 0
+  TTLSEnvironmentAdvancedParmsRef ServerEnvironmentAdvParms
+  TTLSKeyringParmsRef ZoweKeyring
+}
+
+TTLSConnectionAction ZoweServerConnectionAction
+{
+  HandshakeRole ServerWithClientAuth
+  TTLSCipherParmsRef CipherParms
+  TTLSConnectionAdvancedParmsRef ZoweConnectionAdvParms
+}
+```
+
+The `PortRange` of this inbound rule is taken from the list of API Mediation Layer components in the `zowe.yaml` file. The `PortRange` should cover the following components:
+
+| Component | Port |
+|----|-----------------------|
+| Gateway | default port 7554 |
+| Discovery | default port 7553 |
+| Caching Service | default port 7555 |
+| API Catalog | default port 7552 |
+| Metrics Service | default port 7551 |
+| Zowe System Services (ZSS) | default port 7557 |
+| Zowe Application Server | default port 7556 |
+
+__Follow this step:__
+
+Replace `ZoweKeyring` with the keyring configured for your installation. Follow [the SAF keyring instructions](../../getting-started/zowe-certificates-overview.md#saf-keyring) in the article _Zowe Certificates overview_ to configure keyrings for your Zowe instance.
+
+Note the setting `HandshakeRole`. This setting applies to core services which authenticate through certificates with each other. This setting allows the API Gateway to receive and accept X.509 client certificates from API Clients.
+
+For more granularity in the AT-TLS rules, separate the rules that need to support Client Certificate authentication (Discovery Service, Gateway Service) from the rest.
+
+### Outbound rules
+
+Outbound rules in this section allow Zowe services to communicate with each other and to other southbound services using HTTP.
+
+Careful consideration needs to be set especially regarding which rules will be configured to send a Client Certificate. Since this cannot be configured on a per-request basis.
+I.E. Do not configure the rule to send the Zowe Server certificate to the API Gateway or to a southbound service that supports X.509 Client Certificate authentication. This will result in unintentionally authenticating the server ACID.
+
```

Comment:
To disable the client certificate set value `CertificateLabel` to empty value, ie:

```
TTLSConnectionAction ClientConnectionAction
{
  HandshakeRole Client
  TTLSCipherParmsRef CipherParms
  TTLSConnectionAdvancedParmsRef ConnectionAdvancedParms
  CertificateLabel
}
```

Reply:
I already clarified this item in the section about communication with southbound services, but I guess I can add some disclaimer here as well to reinforce it

Reply:
Added example